launch.py

```
@@ -221,95 +221,14 @@ def prepare_environment(args: argparse.Namespace) -> bool:
             return False
         # Download NLTK data immediately after Python deps are installed
         if not args.no_download_nltk:
-            if not download_nltk_data(python_executable):
+            if not download_nltk_data():
```

Pylint (4.0.2)

[error] No value for argument 'python_executable' in function call

(E1120)